include/zephyr/drivers/sensor/ade9153a.h

```diff
@@ -0,0 +1,410 @@
+#define ADE9153A_REG_RUN              0x0480
+/* Configuration Register 1. */
+#define ADE9153A_REG_CONFIG1          0x0481
+/* Time between positive to negative zero crossings on Phase A voltage and current. */
+#define ADE9153A_REG_ANGL_AV_AI       0x0485
+/* Time between positive to negative zero crossings on Phase A and Phase B currents. */
+#define ADE9153A_REG_ANGL_AI_BI       0x0488
+/* Voltage RMS_OC dip detection cycle configuration. */
+#define ADE9153A_REG_DIP_CYC          0x048B
+/* Voltage RMS_OC swell detection cycle configuration. */
+#define ADE9153A_REG_SWELL_CYC        0x048C
+/* CFx configuration register. */
+#define ADE9153A_REG_CFMODE           0x0490
+/* Computation mode register. Set this register to 0x0005. */
+#define ADE9153A_REG_COMPMODE         0x0491
+/* Accumulation mode register. */
+#define ADE9153A_REG_ACCMODE          0x0492
+/* Configuration Register 3 for configuration of power quality settings. */
+#define ADE9153A_REG_CONFIG3          0x0493
+/* CF1 denominator register. */
+#define ADE9153A_REG_CF1DEN           0x0494
+/* CF2 denominator register. */
+#define ADE9153A_REG_CF2DEN           0x0495
+/* Zero-crossing timeout configuration register. */
+#define ADE9153A_REG_ZXTOUT           0x0498
+/* Voltage channel zero-crossing threshold register. */
+#define ADE9153A_REG_ZXTHRSH          0x0499
+/* Zero-crossing detection configuration register. */
+#define ADE9153A_REG_ZX_CFG           0x049A
+/* Power sign register. */
+#define ADE9153A_REG_PHSIGN           0x049D
+/* This register holds the CRC of the configuration registers. */
+#define ADE9153A_REG_CRC_RSLT         0x04A8
+/* The register holds the 16-bit CRC of the data sent out on the MOSI/RX pin during the last SPI
+ * register read.
+ */
+#define ADE9153A_REG_CRC_SPI          0x04A9
+/* This register holds the data read or written during the last 16-bit transaction on the SPI port.
+ * When using UART, this register holds the lower 16 bits of the last data read or write.
+ */
+#define ADE9153A_REG_LAST_DATA_16     0x04AC
+/* This register holds the address and the read/write operation request (CMD_HDR) for the last
+ * transaction on the SPI port.
+ */
+#define ADE9153A_REG_LAST_CMD         0x04AE
+/* Configuration Register 2. This register controls the high-pass filter (HPF) corner and the user
+ * period selection.
+ */
+#define ADE9153A_REG_CONFIG2          0x04AF
+/* Energy and power accumulation configuration. */
+#define ADE9153A_REG_EP_CFG           0x04B0
+/* Power update time configuration. */
+#define ADE9153A_REG_PWR_TIME         0x04B1
+/* Energy accumulation update time configuration. */
+#define ADE9153A_REG_EGY_TIME         0x04B2
+/* This register forces an update of the CRC of configuration registers. */
+#define ADE9153A_REG_CRC_FORCE        0x04B4
+/* Temperature sensor configuration register. */
+#define ADE9153A_REG_TEMP_CFG         0x04B6
+/* Temperature measurement result. */
+#define ADE9153A_REG_TEMP_RSLT        0x04B7U
```

Comment:
@rodrigopex This value is defined with the suffix `U` - differently to the rest of the register addresses in this header file. Is this intentional?

Reply:
I will fix that. Thanks.